### PR TITLE
Add some pre-commit check items.

### LINKED ISSUE #7: Need add CI for the repo

### PR DESCRIPTION
**Pull request by pkuyym**
Resolves #7

### CODE REVIEW

**Comment by pkuyym** [comment]
@wangkuiyi Please help to authorize the repo for CI.

File: .copyright.hook [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+from __future__ import unicode_literals
+
+import argparse
+import io, re
+import sys, os
+import subprocess
+import platform
+
+COPYRIGHT = '''
+  Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```
Comment (kuke) [inline .copyright.hook]: 2018

Comment (pkuyym) [reply .copyright.hook]: clang-format removed.

**Comment by pkuyym** [comment]
I merged this PR to test CI. If necessary, we can revert it.

File: .copyright.hook (hunk 1 above)
Comment (varunarora) [reply .copyright.hook]: Great - yes this is exactly it needs to be dynamically replaced

**Pull request merged**